build.gradle.kts

```diff
@@ -13,22 +13,22 @@ dependencies {
     annotationProcessor("org.springframework.boot:spring-boot-configuration-processor")
 
     runtimeOnly("com.h2database:h2:1.4.200")
-    runtimeOnly("org.flywaydb:flyway-core:6.4.4")
-    runtimeOnly("org.postgresql:postgresql:42.2.14")
+    runtimeOnly("org.flywaydb:flyway-core:6.5.4")
+    runtimeOnly("org.postgresql:postgresql:42.2.15")
 
     implementation("org.springframework.boot:spring-boot-starter-data-jpa")
     implementation("org.springframework.boot:spring-boot-starter-cache")
     implementation("org.springframework.boot:spring-boot-starter-security")
     implementation("org.springframework.boot:spring-boot-starter-oauth2-resource-server")
     implementation("org.springframework.boot:spring-boot-starter-oauth2-client")
 
-    implementation("io.springfox:springfox-swagger2:2.9.2")
-    implementation("io.springfox:springfox-swagger-ui:2.9.2")
+    implementation("io.springfox:springfox-swagger2:3.0.0")
```

Comment:
Some projects had issues with 3.0.0 I think

Reply:
I've updated relevant config to get this to work. It generates random errors around void responses due to this bug : https://github.com/springfox/springfox/issues/3476 but it's only cosmetic.